core/impl/src/main/java/org/eclipse/sensinact/core/command/impl/GatewayThreadImpl.java

```diff
@@ -128,6 +130,24 @@ void removeEPackage(EPackage ePackage) {
         nexusImpl.removeEPackage(ePackage);
     }
 
+    @Reference(service = WhiteboardGetSet.class, cardinality = MULTIPLE, policy = DYNAMIC)
+    void addWhiteboardResourceHandler(WhiteboardGetSet<?> getHandler, Map<String, Object> props) {
+        whiteboard.addWhiteboardHandler(getHandler, props);
+    }
+
+    void removeWhiteboardResourceHandler(WhiteboardGetSet<?> getHandler, Map<String, Object> props) {
+        whiteboard.removeWhiteboardHandler(getHandler, props);
+    }
+
+    @Reference(service = WhiteboardAct.class, cardinality = MULTIPLE, policy = DYNAMIC)
+    void addWhiteboardActHandler(WhiteboardAct<?> actHandler, Map<String, Object> props) {
+        whiteboard.addWhiteboardActHandler(actHandler, props);
+    }
+
```

Comment:
Was there a reason to avoid managing service update?

Reply:
Still a work in progress